### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by dved**
see the latest commit. 
First one was a mistake. Second one was reverting the first one.

parser now works assuming that books.csv contains more recoreds then users.json. It is planned to rewrite it to use generators and don't rely on the number of books

### CODE REVIEW

File: home_work3/src/assign_books.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+import json, csv, argparse;
```
Comment (e7506c) [inline home_work3/src/assign_books.py:1]: тут ; не нужно

File: home_work3/src/assign_books.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+books_keys = [k.lower() for k in books[0]]
+# create list of values
+books_list_values = [books[i] for i in range(1, len(books))]
+# create list of dictionaries from 2 lists
+books_list_dic = [{key: value for key, value in zip(books_keys, books_list_values[i])} for i in range(len(books_list_values))]
+
+book_reduced_keys = ('title', 'author','genre','pages')
+books_reduced_data = [{key: value for key, value in book.items() if key in book_reduced_keys} for book in books_list_dic]
+total_books = len(books_reduced_data)
+if(total_books < total_users):
```
Comment (e7506c) [inline home_work3/src/assign_books.py:63]: скобки тоже не нужны. проверьте, пожалуйста, на PEP8,

File: home_work3/src/assign_books.py (hunk 1 above)
Comment (dved) [reply home_work3/src/assign_books.py:1]: ok, увидел

File: home_work3/src/assign_books.py (hunk 2 above)
Comment (dved) [reply home_work3/src/assign_books.py:63]: наследие других языков

**Pull request merged**